src/android/MigrateStorage.java

```diff
@@ -28,46 +27,51 @@
  */
 public class MigrateStorage extends CordovaPlugin {
     // Switch this value to enable debug mode
-    private static final boolean DEBUG_MODE = false;
+    private static final boolean DEBUG_MODE = true;
```

Comment:
Don't forget to set this to `false` if it doesn't make sense for production build.